src/ucs/datastruct/arbiter.c

```diff
@@ -22,12 +22,10 @@ void ucs_arbiter_group_init(ucs_arbiter_group_t *group)
 
 void ucs_arbiter_cleanup(ucs_arbiter_t *arbiter)
 {
-    ucs_assert(arbiter->current == NULL);
```

Comment:
because we can not guarantee that there will be no pending op during interface cleanup.